src/common/navigation.yaml

```diff
@@ -1,12 +1,14 @@
 sections:
   - pages:
       - path: /
-      - path: /build-an-app # is an overview page
-      - path: /smart-contracts
+      - path: /build-apps # is an overview page
+      - path: /write-smart-contracts
         pages:
           - path: /overview
           - path: /principals
-          - path: /clarity-values
+          - path: /cli-wallet-quickstart
+          - path: /install-source
```

Comment:
These two pages were seemingly never linked to after they were written. Not sure if that was intentional, but I added links to them in this change.

Reply:
linking to them is probably right. I have an issue open to update the CLI quickstart page as well

Reply:
ah, wait - the install source page seems unfinished. wouldn't link to it (and maybe remove the file entirely?)